src/collectors/dropwizard/dropwizard.py

```diff
@@ -32,65 +99,55 @@ def get_default_config(self):
         """
         config = super(DropwizardCollector, self).get_default_config()
         config.update({
-            'host':     '127.0.0.1',
-            'port':     8081,
-            'path':     'dropwizard',
+            'url': DEFAULT_METRICS_URL,
+            'path': 'dropwizard'
         })
         return config
 
     def collect(self):
         if json is None:
             self.log.error('Unable to import json')
             return {}
-        url = 'http://%s:%i/metrics' % (
-            self.config['host'], int(self.config['port']))
+
         try:
-            response = urllib2.urlopen(url)
-        except urllib2.HTTPError as err:
+            response = requests.get(self.config['url'])
+        except requests.exceptions.RequestException as err:
             self.log.error("%s: %s", url, err)
             return
 
+        if response.status_code != 200:
+            self.log.error("Received invalid status code %d",
```

Comment:
Would recommend adding the collector name in the log (to help with debugging) - unless it is inserted automatically.